src/main/java/com/unosquare/cvgenerator/controller/HelloController.java

```diff
@@ -5,17 +5,19 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
-@RequestMapping("/api")
+@RequestMapping("/")
 @RequiredArgsConstructor
 public class HelloController {
 
     private final GreetingService greetingService;
 
+    @CrossOrigin
```

Comment:
will be a bit cumbersome to have to annotate this everywhere - better to investigate a way we can set up CORS using a global config.

Quickly found [this](https://stackoverflow.com/a/51721298/3271729), feel free to investigate different options